config/console/kustomization.yaml

```diff
@@ -0,0 +1,15 @@
+namePrefix: mcg-ms-
+
+# Labels to add to all resources and selectors.
+commonLabels:
+  app: mcg-ms-console
+
+resources:
+- console_init.yaml
+
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+images:
+- name: mcg-ms-console
+  newName: quay.io/skatiyar/mcg-ms-console
```

Comment:
plz let me know and I will push the image to that repo (if there is any common place for development purpose)...
also once build team comes into the picture I think we don't we will have such issues !!

Reply:
I don't think I have access, but I think we should not block this PR on this. We can always change it later.
Or if someone has access they can push the image themselves, we just need:
https://github.com/red-hat-storage/mcg-ms-console/blob/master/Dockerfile
https://github.com/red-hat-storage/mcg-ms-console/blob/master/default.conf 
That is all, it will do the cloning, building and everything. And we can push that to any repository.

Reply:
yes @SanjalKatiyar it's not a blocker, just a suggestion